index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="app-wrap">
+        <header class="app-header">
+            <div class="logo"><p class="text_logo">My Logo</p></div>

```

Comment:
- [ ] Please 🙏🏽 use anchor tag `a` instead of `p` tag for the best practices of HTML & CSS, the logo is a link.